lib/core-classes/AssetService.ts

```diff
@@ -109,11 +98,24 @@ export class AssetService {
   // @todo remove when we have the date extractor in the core
   private iso8601 (value: string, name: string): Date {
     const parsed: any = new Date(value);
-
     if (isNaN(parsed)) {
       throw new BadRequestError(`"${name}" is not a valid ISO8601 date`);
     }
-
     return parsed;
   }
+
+
+  async getDeviceByAsset(assetId :string) {
```

Comment:
Maybe we could add a new field in the Asset document containing the linked devices IDs. It would allows to update them in a easiest way without having to search (and optionally refresh).

What do you think?

Reply:
That would be useful for many different case, I will implement that.